modules/installation-and-upgrade/pages/container-deployment/prepare-micro-host.adoc

```diff
@@ -1,22 +1,28 @@
 == Prepare {sle-micro} {microversion} Host
 
+=== Download the installation media
+
 .Procedure: Downloading the Installation Media
 . Locate the {sle-micro} {microversion} installation media at https://www.suse.com/download/sle-micro/.
 . Download [filename]``SLE-Micro-5.5-DVD-x86_64-GM-Media1.iso``.
 . Prepare a DVD or USB flash drive with the downloaded [filename]``.iso`` image for installation.
-. Insert the DVD or USB flash drive (USB disk or key) containing the installation image for {sle-micro} {microversion}.
-. Boot or reboot your system.
+
+
+=== Install {sle-micro} {microversion}
 
 For more information about preparing your machines (virtual or physical), see link:https://documentation.suse.com/sle-micro/5.5/html/SLE-Micro-all/book-deployment-slemicro.html[{sle-micro} {microversion} Deployment Guide].
 
 .Procedure: Installing {sle-micro} {microversion}
+. Insert the DVD or USB flash drive (USB disk or key) containing the installation image for {sle-micro} {microversion}.
+. Boot or reboot your system.
```

Comment:
I don't believe, this step is needed.

Reply:
But isn't that the way to kick-off the installation (at least on-prem)?

Reply:
ah correct, sorry I was just focusing on reboot part. Yes, you are right.

Reply:
;)